src/js/claims-status/components/AddingDetails.jsx

```diff
@@ -3,8 +3,8 @@ import React from 'react';
 class AddingDetails extends React.Component {
   render() {
     return (
-      <div className="usa-alert usa-alert-info claims-no-icon claims-alert-status">
-        <h4>We’re adding your details</h4>
+      <div className="usa-alert usa-alert-info no-background-image claims-alert-status">
```

Comment:
The original issue recommends using single class selectors as much as possible. Should we consider using `@extend` selectively to achieve this?

Reply:
We generally shouldn't use `@extend`: https://github.com/department-of-veterans-affairs/vets-website/blob/master/docs/CSSandSCSSBestPractices.md#dont-use-extend

Also, to be clear, I don't mean single classes in markup, I mean that when you write a rule in a scss file, we should try to avoid complicated selectors.